backend/docker/sql/views/signed_extrinsics_per_month.sql

```diff
@@ -0,0 +1,6 @@
+CREATE view signed_extrinsics_per_month AS
```

Comment:
Add `view` postfix. `signed_extrinsics_per_month_view`

Reply:
added

Reply:
Add to the view name as well. Not only file.

Reply:
added